File: benchmarks/cross-lang/fib40/zig/main.zig
fn fib(n: i64) i64 {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

pub fn main() u8 {
    const r = fib(40);
    return @intCast(r & 255);
}

File: benchmarks/cross-lang/fib40/rs/main.rs
fn fib(n: i64) -> i64 {
    if n < 2 { n } else { fib(n - 1) + fib(n - 2) }
}

fn main() {
    let r = fib(40);
    std::process::exit((r & 255) as i32);
}
